schema.js

```diff
@@ -6,6 +6,33 @@ const joi = require('joi');
 const levels = Object.values(Severity).filter(level => typeof level === 'string')
   || ['fatal', 'error', 'warning', 'log', 'info', 'debug', 'critical'];
 
+const sentryClient = joi.object().keys({
+  configureScope: joi.func().minArity(1),
+  Scope: joi.func().required(),
+  Parsers: joi.object().keys({
+    parseError: joi.func().minArity(1).required(),
+  }).unknown().required(),
+  Handlers: joi.object().keys({
+    parseRequest: joi.func().minArity(2).required(),
+  }).unknown().required(),
+  withScope: joi.func().minArity(1).required(),
+  captureEvent: joi.func().minArity(1).required(),
+}).unknown();
+
+const sentryOptions = joi.object().keys({
+  dsn: joi.string().uri().allow(false).required(),
+  debug: joi.boolean(),
+  release: joi.string(),
+  environment: joi.string(),
+  sampleRate: joi.number().min(0).max(1),
+  maxBreadcrumbs: joi.number().integer(),
+  attachStacktrace: joi.any(),
+  sendDefaultPii: joi.boolean(),
+  serverName: joi.string(),
+  beforeSend: joi.func(),
+  beforeBreadcrumb: joi.func(),
+});
```

Comment:
i don't think it's our job to do validation for sentry. sentry should do that internally.

imo: just check for dsn and allow unknown.

Reply:
(i know this is not part of this PR. But i just noticed this now :sweat_smile:)